apps/authentication/mfa/face.py

```diff
@@ -30,10 +30,11 @@ def is_active(self):
 
     @staticmethod
     def global_enabled():
-        return settings.XPACK_LICENSE_IS_VALID \
-            and LicenseEditionChoices.ULTIMATE == \
-            LicenseEditionChoices.from_key(settings.XPACK_LICENSE_EDITION) \
-            and settings.FACE_RECOGNITION_ENABLED
+        return (
+            settings.XPACK_LICENSE_IS_VALID and
+            settings.XPACK_LICENSE_EDITION_ULTIMATE and
+            settings.FACE_RECOGNITION_ENABLED
+        )
 
     def get_enable_url(self) -> str:
         return '/ui/#/profile/index'
```

Comment:
As of 2023, this code snippet doesn't appear to have any significant discrepancies or issues that could be corrected through a knowledge update. However, it does seem that some context elements like `settings.XPACK_LICENSE_EDITION`, `LicenseEditionChoices` (which might relate specifically to the licenses or editions), might need updates due to changes in software configurations.

The only issue seems to be the use of Python list concatenation directly after constructing a boolean condition; using f-strings (`format`) instead would typically offer more readability and maintainability:

```python
@staticmethod
def global_enabled():
-    return settings.XPACK_LICENSE_ISVALID and  
+    return (
+        settings.XPACK_LICENSING_VALID and
+        settings.XPACK_LICENSES_ULTIMATE and
+        settings.FACE_RECOGNITION_ENABLED
+    )
```

Overall, I do not see anything urgent within current limitations but suggest looking into these details before moving forward for enhancements or new projects based on what you are currently planning with this codebase.